lib/build/bundle.js

```diff
@@ -108,7 +108,7 @@ exports.Bundle = class {
 
     if (loaderOptions.configTarget === this.config.name) {
       work = work.then(() => {
-        files.push({ contents: this.writeLoaderConfig(platform, loaderOptions) });
+        files.push({ contents: "_loadAureliaRequireJSConfig();"});
```

Comment:
I wanted it to mention something about Aurelia at least. Since this is being inserted in the vendor bundle file, it is possible (unlikely and they shouldn't do it anyway) that somebody define in the global namespace a similar function called `_configureModuleLoader`. So by naming it something like `_aureliaConfigureModuleLoader`, or something like that, we're better ensuring that people aren't going to override our function.